AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -112,6 +112,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+debrid-link.fr###cookies_infos
```

Comment:
The indent at the top remains. Please use this rule:
`debrid-link.fr#%#//scriptlet('set-cookie', 'cookies_infos', '1')`

Btw you can commit in the current branch and the current PR.

Reply:
Thank you for tell me.

Reply:
@SKEIDs 
I moved the rule to the rest of JS-rules, please take a look.